addons/stock/report/report_deliveryslip.py

```diff
@@ -0,0 +1,15 @@
+from odoo import api, models
+
+
+class ReportStockPickingDeliverySlip(models.AbstractModel):
+    _name = 'report.stock.report_deliveryslip'
+    _description = 'Stock delivery slip report'
+
+    @api.model
+    def _get_report_values(self, docids, data=None):
+        return {
+            'doc_ids': docids,
+            'doc_model': self.env['stock.picking'],
+            'data': data,
+            'docs': self.env['stock.picking'].browse(docids),
+        }
```

Comment:
It seems like this is a copy of the existing default behavior, so do you actually need to add this into stock for any reason or can you just declare it in the pt localization?